docs/source/noise/index.rst

```diff
@@ -0,0 +1,96 @@
+.. _noise_main:
+
+======
+ Noise
+======
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: entry
+
+In order to have a realistic challenge with entity resolution, it is essential
+to add noise to the data. Noise can also be called "corruption" or "distortion."
+This page describes the types of noise included in Pseudopeople.
+
+Categories of noise
+-------------------
+
+Pseudopeople can add two broad categories of noise to the datasets it generates:
+
+#. **Row-based noise:** Errors in the inclusion or exclusion of entire rows of
+   data, such as duplication or omission
+#. **Column-based noise:** Errors in data entry for individual fields within a
+   row, such as miswriting or incorrectly selecting responses
+
+Each type of row-based noise operates on the entire dataset (selecting rows to
+include or exclude), while each type of column-based noise operates on one
+column of data at a time (selecting cells within that column to noise).
+Pseudopeople applies row-based noise prior to column-based noise so that errors
+in individual fields are applied to the records (rows) that actually appear in
+the data. Each type of column-based noise is applied independently to a subset
+of the columns in the dataset, where each column represents the values of a
+single field for all the records.
+
+Customizing the noise in a dataset
+----------------------------------
+
+Pseudopeople has a configuration system to specify which types of noise are
+applied to each dataset, which columns each type of column-based noise is
+applied to, and any additional parameters needed by the noise functions.
```

Comment:
I don't think you can change which columns they apply to, can you?

But in any case I think you could say that by just adding "and column" to the first clause in this sentence. I know that it wouldn't be technically correct in the case of row noise types, but I don't think it's important to be that exact here.